css/css-scroll-snap/scroll-target-snap-002.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and snap position with snapping on (y-axis)</title>
+<link rel='author' title='Elika J. Etemad' href='http://fantasai.inkedblade.net/contact'>
+<link rel='help' href='https://www.w3.org/TR/css-scroll-snap-1/#choosing'>
+<link rel='match' href='scroll-target-001-ref.html'>
+
+<style type='text/css'>
+  .container {
+    border: solid blue 4px;
+    height: 4em;
+    overflow: auto;
+    scroll-snap-type: block;
+
+    /* to make failing more obvious */
+    background: 0 1em / 100% 1em linear-gradient(red, red) repeat-x;
+    /* avoid anti-aliasing issues */
+    font: 20px/1 sans-serif;
+  }
+  .container > div {
+    height: 1em;
+  }
+  #target    { scroll-margin:    1em 0 0;
```

Comment:
Shouldn't that be `scroll-margin:    0 0 1em` ?

Reply:
No, the target element is in the third quarter, not the second quarter.